packages/stems/src/components/Button/Button.module.css

```diff
@@ -32,6 +32,25 @@
   height: var(--unit-8);
 }
 
+.button.large {
```

Comment:
cc @dylanjeffers For the button updates. I tried to make the changes in a way that would only have a major effect if you used the new `LARGE` size. It's a reasonable starting point for getting buttons migrated over to the new harmony standards. Happy to chat more about how we could clean this up. But it's a much bigger audit to check all the places we use buttons 😅

Reply:
cc @sliptype